packages/@react-stately/table/src/useTableState.ts

```diff
@@ -35,13 +37,16 @@ export interface TableState<T> extends GridState<T, ITableCollection<T>> {
 
 export interface CollectionBuilderContext<T> {
   showSelectionCheckboxes: boolean,
+  showDragButtons: boolean,
   selectionMode: SelectionMode,
   columns: Node<T>[]
 }
 
 export interface TableStateProps<T> extends CollectionStateBase<T, ITableCollection<T>>, MultipleSelectionStateProps, Sortable {
   /** Whether the row selection checkboxes should be displayed. */
-  showSelectionCheckboxes?: boolean
+  showSelectionCheckboxes?: boolean,
+  /** Whether the row drag button should be displayed. */
+  showDragButtons: boolean
```

Comment:
ListView doesn't have this, any chance we can drop this from the props?

Reply:
I think this is necessary so that we know to include a cell for the drag handle when building the table collection. Open to suggestions if there is a way to go without it though.

Reply:
ahhh

we just need the same logic that we have in TableView. we're already passing all the props through to useTableState

so just need this
```
    dragAndDropHooks
  } = props;
  let isTableDraggable = !!dragAndDropHooks?.useDraggableCollectionState;
```

then we shouldn't need a special prop